core/templates/dev/head/services/ConstructTranslationIdsService.js

```diff
@@ -16,14 +16,16 @@
  * @fileoverview Service to dynamically construct translation ids for i18n.
  */
 
-oppia.factory('ConstructTranslationIdsService', function() {
-  return {
+oppia.factory('ConstructTranslationIdsService', [
+  function() {
+    return {
     // Construct a translation id for library from name and a prefix.
     // Ex: 'categories', 'art' -> 'I18N_LIBRARY_CATEGORIES_ART'
-    getLibraryId: function(prefix, name) {
-      return (
-        'I18N_LIBRARY_' + prefix.toUpperCase() + '_' +
+      getLibraryId: function(prefix, name) {
+        return (
+          'I18N_LIBRARY_' + prefix.toUpperCase() + '_' +
         name.toUpperCase().split(' ').join('_'));
```

Comment:
Indent by 4 (this should be 2 spaces indented from the previous line). In the case that the linter suggests otherwise, go with the linter.

Reply:
Done, thanks!